wifiphisher/common/interfaces.py

```diff
@@ -387,6 +387,24 @@ def internet_access_enable(self, value):
         else:
             raise InvalidValueError(value, bool)
 
+    def nm_unmanage(self, interface):
+        """
+        Set an interface to unmanaged.
+
+        :param interface: Name of the interface
+        :type interface: str
+
+        :return: True upon success
+        :rtype bool
+        """
+
+        out = check_output(["nmcli", "dev",  interface, "managed", "no"])
```

Comment:
`check_output` needs to be in a `try` `except` in case non-zero return value. Also `stderr` needs to be handled as well.